# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../flex/flex.skl-fix.patch
# Copyright (C) 2004 - 2005 The T2 SDE Project
# Copyright (C) 1998 - 2004 ROCK Linux Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---

--- flex-2.5.31/flex.skl.old	2004-01-01 20:38:27.000000000 +0100
+++ flex-2.5.31/flex.skl	2004-01-01 20:38:49.000000000 +0100
@@ -478,7 +478,7 @@
 
 m4_ifdef( [[M4_YY_NOT_IN_HEADER]],
 [[
-#define unput(c) yyunput( c, YY_G(yytext_ptr) M4_YY_CALL_LAST_ARG )
+#define unput(c) yyunput( c, YY_G(yytext) M4_YY_CALL_LAST_ARG )
 ]])
 
 /* The following is because we cannot portably get our hands on size_t
